systatus.go

```diff
@@ -88,20 +91,29 @@ func handleMem(w http.ResponseWriter, r *http.Request) {
 func handleDisk(w http.ResponseWriter, r *http.Request) {
 
 }
-func handleEnv(w http.ResponseWriter, r *http.Request) {
-
+func handleEnv(w http.ResponseWriter, r *http.Request, sensitiveKeys []string) {
 	res := EnvResponse{}
 	env := os.Environ()
 
 	res.Env = make(map[string]string, len(env))
 
+	sensitiveKeysMap := make(map[string]bool)
+	for _, key := range sensitiveKeys {
+		sensitiveKeysMap[key] = true
+	}
+
 	for _, val := range env {
 		split := strings.Split(val, "=")
-		res.Env[split[0]] = split[1]
+		key := split[0]
+		value := split[1]
+
+		if sensitiveKeysMap[key] {
+			value = "********"
+		}
```

Comment:
I was wondering if we can directly check `sensitiveKeys` param instead of using a support hash map